vision_evaluation/evaluators.py

```diff
@@ -1100,3 +1100,46 @@ def add_predictions(self, predictions, ground_truths):
 
     def get_report(self, **kwargs):
         return {self._get_id(): {group: eval.get_report(**kwargs) for group, eval in self._evaluators.items()}}
+
+
+class MeanLpErrorEvaluator(Evaluator):
+    """
+    Error evaluator for regression targets
+    Parameters:
+        p: Lp order of the norm used
+    """
+
+    def __init__(self, p: int):
```

Comment:
you have a test of p = 0.5

Reply:
Yes, theoretically that could be an Lp norm that someone may want to calculate, so I was thinking to leave that option open? Your thoughts?

Reply:
Test case removed since it's not commonly used.